Update site-wide-analysis.md (#9378)
magento/devdocs: src/tools/site-wide-analysis.md

#### Service

> 1. Create a systemd unit file (`/etc/systemd/system/scheduler.service`) with the following configuration. (Replace `<filesystemowner>` with unix user owner of /path/to/agent directory)

Reviewer: ```suggestion
1. Create a systemd unit file (`/etc/systemd/system/scheduler.service`) with the following configuration (replace `<filesystemowner>` with the Unix user that owns the directory where the agent is installed).
```
